adal/src/main/java/com/microsoft/aad/adal/AcquireTokenSilentHandler.java

```diff
@@ -169,14 +170,23 @@ void setWebRequestHandler(final IWebRequestHandler webRequestHandler) {
      * Attempt to get new access token with regular RT. 
      */
     private AuthenticationResult tryRT() throws AuthenticationException {
+        CacheEvent cacheEvent = new CacheEvent(EventStrings.TOKEN_CACHE_LOOKUP);
+        cacheEvent.setTokenType(EventStrings.TOKEN_TYPE_RT);
+        cacheEvent.setRequestId(mAuthRequest.getTelemetryRequestId());
+        Telemetry.getInstance().startEvent(mAuthRequest.getTelemetryRequestId(), EventStrings.TOKEN_CACHE_LOOKUP);
```

Comment:
I meant put the cache event inside getRegularTokenCacheItem, getMRRTItem, ect.

---
In reply to: [76135670](https://github.com/AzureAD/azure-activedirectory-library-for-android/pull/703#discussion_r76135670) [](ancestors = 76135670)